src/kata6.joinNames.js

```diff
@@ -1,5 +1,29 @@
 const joinNames = (namesObj) => {
+    const nameArray = namesObj.map(function(x) {
+       return x.name
+    });
 
-}
+    for (let i = 0; i < nameArray.length; i++) {
```

Comment:
No need for line breaks after blocks open or before they close. This is very spacey! 😄